Formula/rust.rb

```diff
@@ -101,6 +102,13 @@ def install
     rm_rf prefix/"lib/rustlib/install.log"
   end
 
+  def post_install
+    Dir["#{lib}/rustlib/**/*.dylib"].each do |dylib|
+      chmod 0755, dylib
```

Comment:
I don't see any particular harm in leaving them like this, although it might be good to change them back from a consistency standpoint.